src/java.net.http/share/classes/jdk/internal/net/http/HttpClientImpl.java

```diff
@@ -1686,6 +1690,14 @@ BufferSupplier getSSLBufferSupplier() {
         return sslBufferSupplier;
     }
 
+    private Duration getIdleConnectionTimeoutProp() {
+        // Http 2 in prop name somewhere
+        String s = Utils.getNetProperty("jdk.httpclient.idleConnectionTimeout");
```

Comment:
Is this new system property expected/allowed to be configured even in `net.properties` file? The implementation of `Utils.getNetProperty` will check even the `net.properties` file for this value, so just checking if that's what we want. If we do want this to be configurable in `net.properties`, should we updated that file to have a commented section which this property and some documentation about it?

Furthermore, I think we should read this value just once and cache the `Duration` as a static member of this class instead of reading this more than once.

Reply:
Good point. TBD whether we want this in `net.properties`. We might - but then it's true that if we do we should have some documentation there.